### PR TITLE
Adding hdfs-conf.yaml to all test cases to support security.

### PR DESCRIPTION
**Pull request by csivaguru**

### CODE REVIEW

**Comment by harshach** [comment]
+1

File: external/storm-hdfs/src/test/java/org/apache/storm/hdfs/trident/TridentFileTopology.java [hunk 1]
```diff
@@ -75,14 +78,20 @@ public static StormTopology buildTopology(String hdfsUrl){
...
             cluster.submitTopology("wordCounter", conf, buildTopology(args[0]));
             Thread.sleep(120 * 1000);
-        }
-        else if(args.length == 2) {
+        } else if(args.length == 3) {
             conf.setNumWorkers(3);
-            StormSubmitter.submitTopology(args[1], conf, buildTopology(args[0]));
+            StormSubmitter.submitTopology(args[2], conf, buildTopology(args[0]));
         } else{
             System.out.println("Usage: TridentFileTopology <hdfs url> [topology name]");
```
Comment (HeartSaVioR) [inline external/storm-hdfs/src/test/java/org/apache/storm/hdfs/trident/TridentFileTopology.java]: @csivaguru You need to update usage here as well.

File: external/storm-hdfs/src/test/java/org/apache/storm/hdfs/trident/TridentSequenceTopology.java [hunk 2]
```diff
@@ -73,14 +75,20 @@ public static StormTopology buildTopology(String hdfsUrl){
...
             cluster.submitTopology("wordCounter", conf, buildTopology(args[0]));
             Thread.sleep(120 * 1000);
-        }
-        else if(args.length == 2) {
+        } else if(args.length == 3) {
             conf.setNumWorkers(3);
-            StormSubmitter.submitTopology(args[1], conf, buildTopology(args[0]));
+            StormSubmitter.submitTopology(args[2], conf, buildTopology(args[0]));
         } else{
             System.out.println("Usage: TridentFileTopology <hdfs url> [topology name]");
```
Comment (HeartSaVioR) [inline external/storm-hdfs/src/test/java/org/apache/storm/hdfs/trident/TridentSequenceTopology.java]: @csivaguru same here.

**Comment by HeartSaVioR** [comment]
+1 once my comments are addressed.

**Comment by csivaguru** [comment]
Can this be merged in? @harshach @HeartSaVioR

**Comment by HeartSaVioR** [comment]
+1 Thanks @csivaguru.

**Comment by HeartSaVioR** [comment]
Sorry I need to revert merging this and set commit message to contain STORM-1924.
Sign-off information is overwritten to me (eac9641469b79251f5146be4dcf8e52e6914448f), so please feel free to let me rearrange if you mind. 
Thanks again @csivaguru.

**Comment by HeartSaVioR** [comment]
Merged (cherry-pick) to master, 1.x, 1.0.x branches.

**Pull request merged**